modules/KoreBuild.Tasks/module.targets

```diff
@@ -107,6 +106,19 @@ dependencies.props file in the current project to use latest versions.
       LineupDependenciesFile="$(LineupDependenciesFile)" />
   </Target>
 
+<!--
+####################################################################################
+Target: InstallToolsets
+
+Installs all VS components specified in the korebuild.json required components.
+####################################################################################
+-->
+  <Target Name="InstallToolsets">
+    <InstallToolsets ConfigFile="$(KoreBuildConfigFile)"
+      Quiet="$(Quiet)"
```

Comment:
Consider giving these properties to something more specific like "ToolsetInstallerLogLevel" and "UpgradeToolsets". Because MSBuild is like one big C program, it's good to give properties specific names to reduce the chance of conflict with other targets.